### PR TITLE
Complete JavaDoc of all annotations.

### PR DESCRIPTION
**Pull request by JunyiYi**
Documentation, documentation and documentation.

### CODE REVIEW

**Comment by pragnagopa** [comment]
@asavaritayal - Can you please verify the docs?

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java [hunk 1]
```diff
@@ -12,23 +12,63 @@
...
     String authToken();
 
+    /**
+     * Defines the target of Twilio SMS to which to write.
+     * @return The Twilio SMS target string.
+     */
     String to();
 
+    /**
+     * Defines the source of Twilio SMS to which to write.
```
Comment (metacpp) [inline src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: from which ?

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java [hunk 2]
```diff
@@ -12,23 +12,63 @@
...
+     * @return The Twilio SMS source string.
+     */
     String from();
 
+    /**
+     * Defines the content body of Twilio SMS to which to write.
```
Comment (metacpp) [inline src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: remove `to which`

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java [hunk 3]
```diff
@@ -12,23 +12,63 @@
...
+     * </ul>
+     * @return The dataType which will be used by the Functions runtime.
+     */
     String dataType() default "";
 
+    /**
+     * Defines the account SID of Twilio SMS to which to write.
+     * @return The Twilio SMS account SID string.
+     */
     String accountSid();
```
Comment (metacpp) [inline src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: SID or Sid?

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java [hunk 4]
```diff
@@ -12,23 +12,63 @@
...
+    /**
+     * The variable name used in function.json.
+     * @return The variable name used in function.json.
+     */
     String name();
 
+    /**
+     * <p>Defines how Functions runtime should treat the parameter value. Possible values are:</p>
+     * <ul>
+     *     <li>"" or string: treat it as a string whose value is serialized from the parameter</li>
```
Comment (metacpp) [inline src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: Why it's inconsistent with TimerTrigger.java even the declaration is the same?

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java (hunk 3 above)
Comment (JunyiYi) [reply src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: Sid is used in code, and SID is used in documentation like https://support.twilio.com/hc/en-us/articles/223136607-What-is-an-Application-SID-.

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java (hunk 1 above)
Comment (JunyiYi) [reply src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: The "From" number displayed in the SMS.

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java (hunk 2 above)
Comment (JunyiYi) [reply src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: Done.

File: src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java (hunk 4 above)
Comment (JunyiYi) [reply src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java]: The definition is slightly different between Trigger, Input and Output.

**Comment by pragnagopa** [comment]
Thanks all. @JunyiYi - Please go ahead and merge this PR when it's ready.

**Pull request merged**